fbx_document.cpp

```diff
@@ -3565,7 +3451,7 @@ void FBXDocument::_convert_animation(Ref<FBXState> p_state, AnimationPlayer *p_a
 					for (int32_t value_i = 0; value_i < key_count; value_i++) {
 						weight.values.write[value_i] = animation->track_get_key_value(shape_track_i, value_i);
 					}
-					track.weight_tracks.push_back(weight);
+					// track.weight_tracks.push_back(weight);
```

Comment:
Is this a bug?

Reply:
Those are references to dead code, I would have removed the whole line but wanted to leave it obvious that probably all the surrounding code needs to be removed (or refactored, if we want to keep that functionality)

Reply:
The linter doesn't like dead code. So I'll try to compile.